example/gluon/image_classification.py

```diff
@@ -64,6 +65,7 @@
 parser.add_argument('--kvstore', type=str, default='device',
                     help='kvstore to use for trainer/module.')
 parser.add_argument('--log-interval', type=int, default=50, help='Number of batches to wait before logging.')
+parser.add_argument('--profile', action='store_true')
```

Comment:
Could you add some description for this arg?

Reply:
Gladly